app/assets/stylesheets/src/career_site/_main.scss

```diff
@@ -36,9 +36,9 @@ footer {
   a,
   span,
   div {
-    color: $gray-600;
+    color: $gray-500;
```

Comment:
Ilya already used `var(--tblr-secondary);` [here](https://github.com/freeats/freeats/pull/166/files)

Reply:
may we unite this?